platforms/documentation/docs/src/snippets/java-platform/dep-man/groovy/platform/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'java-platform'
+}
+
+javaPlatform {
+    allowDependencies()
+}
+
+dependencies {
+    constraints {
+        api 'org.apache.commons:commons-lang3:3.12.0'
+        api 'com.google.guava:guava:30.1.1-jre'
+        api 'org.slf4j:slf4j-api:1.7.30'
+    }
+}
```

Comment:
```suggestion
plugins {
    id("java-platform")
}

dependencies {
    constraints {
        api("org.apache.commons:commons-lang3:3.12.0")
        api("com.google.guava:guava:30.1.1-jre")
        api("org.slf4j:slf4j-api:1.7.30")
    }
}
```

Reply:
Updated to use consistent syntax. 

Also removed `allowDependencies`. This is a "power-user" API and should not be needed here.